services/s3/src/main/java/software/amazon/awssdk/services/s3/presigner/S3Presigner.java

```diff
@@ -512,6 +513,8 @@ default PresignedAbortMultipartUploadRequest presignAbortMultipartUpload(
     @SdkPublicApi
     @NotThreadSafe
     interface Builder extends SdkPresigner.Builder {
+        Builder s3Configuration(S3Configuration s3Configuration);
```

Comment:
Can we call this `serviceConfiguration` to match `S3BaseClientBuilder#serviceConfiguration`?

Reply:
Yes, I'll update this